deployments/examples/cs3_users_ocis/ocis/config/proxy-config.json

```diff
@@ -0,0 +1,94 @@
+{
+  "HTTP": {
+    "Namespace": "com.owncloud"
+  },
+  "oidc": {
+    "issuer": "https://localhost:9200",
+    "insecure": true
+  },
+  "policy_selector": {
+    "static": {
+      "policy": "reva"
+    }
+  },
+  "policies": [
+    {
+      "name": "reva",
+      "routes": [
+        {
+          "endpoint": "/",
+          "backend": "http://localhost:9100"
+        },
+        {
+          "endpoint": "/.well-known/",
+          "backend": "http://localhost:9130"
+        },
+        {
+          "endpoint": "/konnect/",
+          "backend": "http://localhost:9130"
+        },
+        {
+          "endpoint": "/signin/",
+          "backend": "http://localhost:9130"
+        },
+        {
+          "type": "regex",
+          "endpoint": "/ocs/v[12].php/cloud/user/signing-key",
+          "backend": "http://localhost:9110"
+        },
+        {
+          "endpoint": "/ocs/",
+          "backend": "http://localhost:9140"
+        },
+        {
+          "endpoint": "/remote.php/",
+          "backend": "http://localhost:9140"
+        },
+        {
+          "endpoint": "/dav/",
+          "backend": "http://localhost:9140"
+        },
+        {
+          "endpoint": "/webdav/",
+          "backend": "http://localhost:9140"
+        },
+        {
+          "endpoint": "/status.php",
+          "backend": "http://localhost:9140"
+        },
+        {
+          "endpoint": "/index.php/",
+          "backend": "http://localhost:9140"
+        },
+        {
+          "endpoint": "/data",
+          "backend": "http://localhost:9140"
+        },
+        {
+          "endpoint": "/api/v0/accounts",
+          "backend": "http://localhost:9181"
```

Comment:
How does this play together with `PROXY_ACCOUNT_BACKEND_TYPE` ?

Reply:
Did you change something on the proxy-config.json compared to the default configuration?

Reply:
Yes, the ocs regex route is different. Removed references to accounts service now.